spec/controllers/accounts_controller_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+describe AccountsController, type: :controller do
+  describe "GET #index" do
+    let! (:accounts) {FactoryGirl.create_list(:account, 20)}
```

Comment:
I think that you don't need the `FactoryGirl.` here if you are including it inside rails_helper